test/test_jit.py

```diff
@@ -1210,7 +1210,51 @@ def get_forward(m):
                    .check_next("aten::int_repr") \
                    .check_next("aten::_dequantize_linear") \
                    .check("return") \
-                   .run(str(m._c._get_method('forward').graph))
+                   .run(str(get_forward(m).graph))
+
+    def test_quant_fusion(self):
+        class Observer(torch.nn.Module):
+            def __init__(self):
+                super(Observer, self).__init__()
+
+            def forward(self, x):
+                return x
+
+            @torch.jit.export
+            def calculate_qparams(self):
+                return torch.tensor([2.0]), torch.tensor([3])
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 5, 3).float()
+
+            def forward(self, x):
+                return self.conv(x)
+
+        def get_forward(m):
+            return m._c._get_method('forward')
+        m = torch.jit.script(M())
+        observer = torch.jit.script(Observer())
+        torch._C._jit_pass_constant_propagation(get_forward(m).graph)
+        m._c = torch._C._jit_pass_prepare_quant(m._c, "forward",
```

Comment:
We don't need to run all the previous passes to test this one. Just explicitly write out the IR we expect on the input and run only this pass on it. The test checking that all passes can work together is also needed, but it's not a unit test (and I think we already have such a test somewhere else, right?).